Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:stretch-slim
+
+RUN apt-get -q -y update && \
+    apt-get install -y --no-install-recommends \
+    tar \
+    rsync \
+    ca-certificates \
+    ssh \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /backup-utils
+ADD https://github.com/github/backup-utils/archive/stable.tar.gz /
```

Comment:
Any reason not to use the deb we create with each release?

Reply:
This way we don’t need to manually bump the version in the Dockerfile each release. Had it using the deb before but @rubiojr suggested using the stable tar for this reason.

Reply:
We could later add pinned versions, but let's see if there is any demand first. There is a bit more maintainer overhead to keep up with versions.

Reply:
:+1: Makes sense to me.